VariableSubstiution.md

```diff
@@ -0,0 +1,27 @@
+# Variable Substitution
```

Comment:
I think the name of this file has a typo

Reply:
renamed file to SimpleTemplating